### PR TITLE
VACMS-1164: Adding wysiwyg replacement field to config. Step 1 of 4.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
See #939 / #1164 

## Testing done
Visual

## Screenshots
image
image

## QA steps
 - [x] As an admin, go to `admin/structure/types/manage/page/form-display` & `admin/structure/types/manage/page/display` & visually verify that new intro text field is not displayed

### CODE REVIEW

**Comment by ethanteague** [comment]
> @ethanteague these look fine... but should't I be seeing in the config that alimited html filter is being applied?

:+1: updated, @swirtSJW

File: config/sync/editor.editor.rich_text_limited.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            - Italic
+        -
+          name: Links
+          items:
+            - DrupalLink
+            - DrupalUnlink
+        -
+          name: Lists
+          items:
+            - BulletedList
```
Comment (swirtSJW) [inline config/sync/editor.editor.rich_text_limited.yml:28]: image

I am seeing settings that are out of scope, unless the requirements changed.  Offering options in the editor that are not supported by the filter I think will be a confusing user experience.

File: config/sync/editor.editor.rich_text_limited.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+            - NumberedList
+        -
+          name: Media
+          items:
+            - Blockquote
```
Comment (swirtSJW) [inline config/sync/editor.editor.rich_text_limited.yml:33]: out of scope?

File: config/sync/editor.editor.rich_text_limited.yml [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+            - Source
+  plugins:
+    language:
+      language_list: un
+    stylescombo:
+      styles: ''
+    drupallink:
+      linkit_enabled: false
+      linkit_profile: ''
+image_upload:
```
Comment (swirtSJW) [inline config/sync/editor.editor.rich_text_limited.yml:47]: images?

File: config/sync/editor.editor.rich_text_limited.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/editor.editor.rich_text_limited.yml]: :+1:  @swirtSJW

File: config/sync/editor.editor.rich_text_limited.yml (hunk 3 above)
Comment (ethanteague) [reply config/sync/editor.editor.rich_text_limited.yml:33]: :+1:  @swirtSJW

File: config/sync/editor.editor.rich_text_limited.yml (hunk 1 above)
Comment (ethanteague) [reply config/sync/editor.editor.rich_text_limited.yml]: :+1:  @swirtSJW
